### PR TITLE
timbot: Bump the 99 weight probability.

### PR DESCRIPTION
**Pull request by timflannagan**
Everyone's favorite restaurant appears to only have a 5% chance of being picked, which is obviously not ideal.

### CODE REVIEW

**Comment by Daniel-Pivonka** [comment]
we might want to bump that probability even higher

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]
Alt Text

**Comment by timflannagan** [comment]
Please take a look at the second commit, which is another implementation I added that bumps the 99 probability even higher!

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]
Failed linting

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]
@Daniel-Pivonka close this monstrosity

**Comment by timflannagan** [comment]
@nathan-weinberg all checks have passed and you're not a main contributor to this project. @Daniel-Pivonka please review and approve so we can get this deployed asap.

**Comment by Daniel-Pivonka** [comment]
@timflannagan1 what happened! you just needed to adjust the probability

File: timbot.py [hunk 1]
```diff
@@ -63,10 +62,7 @@ def uploadimage(path, title, text):
 
 
 def choose_lunchcation():
-    places = ['pauls', 'asian plus', 'moes', 'the 99', 'chilis']
```
Comment (Daniel-Pivonka) [inline timbot.py:66]: we need all this put back!

**Review by Daniel-Pivonka — CHANGES_REQUESTED** [review]
your

File: timbot.py (hunk 1 above)
Comment (timflannagan) [reply timbot.py:66]: Nope, it's not needed if we always choose the 99 as the lunch location.

**Comment by timflannagan** [comment]
On second thought, I'm going to close this as I realize that the 99 is a trash restaurant.

**Pull request closed without merging**